app/dapp/index.js

```diff
@@ -1,5 +1,5 @@
+/* globals global, process */
```

Comment:
is there a way to define globals, ahem, globally?

Reply:
There is, we're using the `globals` package for this.  

https://github.com/floating/frame/pull/1346/files#diff-9601a8f6c734c2001be34a2361f76946d19a39a709b5e8c624a2a5a0aade05f2R55

However I thought this file is renderer process only and 'global' (as opposed to 'globalThis') and 'process' are only available in the node environment, we are only setting browser (see link above).  I logged them out and they do exist at runtime, though.  Basically I got confused and decided to exempt them here...